### ISSUE TITLE
NTR: alpha1 antichymotrypsin low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8337]
New term request alpha1 antichymotrypsin low in blood (elthat_130429170927):

**Comment by pnrobinson** [comment]
@MickeySegal
This is 
[Reduced serum alpha-1-antitrypsin HP:0032025](https://hpo.jax.org/app/browse/term/HP:0032025)

Note we have been moving towards unifying primary labels to Reduced circulating alpha-1-antitrypsin concentration and will do this for this term as well!

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/107280 makes it sound like antichymotrypsin and antitrypsin are different "Alpha-1-antichymotrypsin is a plasma protease inhibitor synthesized in the liver. It is a single glycopeptide chain of about 68,000 daltons and belongs to the class of serine protease inhibitors. In man, the normal serum level is about one-tenth that of alpha-1-antitrypsin (PI; [107400](https://www.omim.org/entry/107400)), with which it shares nucleic acid and protein sequence homology".
